example_configs/kafka-connect.yml

```diff
@@ -0,0 +1,100 @@
+lowercaseOutputName: true
+rules:
+  #kafka.connect:type=app-info,client-id="{clientid}"
+  #kafka.consumer:type=app-info,client-id="{clientid}"
+  #kafka.producer:type=app-info,client-id="{clientid}"
+  - pattern: 'kafka.(.+)<type=app-info, client-id=(.+)><>([a-z-]+): (.+)'
+    name: kafka_$1_appinfo
+    value: 1
+    labels:
+      client: "$2"
```

Comment:
What's this client id?

Reply:
Oh! Yes, it's clientId. I'll fix it.

Reply:
I mean more what is this label value? What is its cardinality?